packages/rocket-auth-aws-infrastructure/README.md

```diff
@@ -0,0 +1,382 @@
+# Authentication Booster Rocket for AWS
+
+This package is a configurable Booster rocket to add an authentication api based on Cognito to your Booster applications.
+
+## Usage
+
+Install this package as a dev dependency in your Booster project (It's a dev dependency because it's only used during deployment, but we don't want this code to be uploaded to the project lambdas)
```

Comment:
```suggestion
Install this package as a `devDependency` in your Booster project (It's a `devDependency` because it's only used during deployment, so we don't want this code to be uploaded to the project lambdas)
```